clifford/_multivector.py

```diff
@@ -739,6 +743,50 @@ def normalInv(self, check=True) -> 'MultiVector':
         """
         return self._pick_inv(fallback=False if check else None)
 
+    def hitzer_inverse(self):
+        """
+        Performs the inversion operation as described in the paper :cite:`Hitzer_Sangwine_2017`
+        """
+        tot = sum(self.layout.sig != 0)
+        if tot <= 5:
+            if tot == 1:
+                # Equation 4.3
+                mv_invol = self.gradeInvol()
+                numerator = mv_invol
+                denominator = (self * mv_invol)[()]
+            elif tot == 2:
+                # Equation 5.5
+                mv_conj = self.conjugate()
+                numerator = mv_conj
+                denominator = (self * mv_conj)[()]
+            elif tot == 3:
+                # Equation 6.5
+                mv_reverse = ~self
+                mv_invol = self.gradeInvol()
+                mv_conj = self.conjugate()
+                numerator = (mv_conj * mv_invol * mv_reverse)
+                denominator = (self * numerator)[()]
```

Comment:
Can make this look a lot more like the case below:
```suggestion
                # Equation 6.5, without the rewrite from 6.4
                mv_conj = self.conjugate()
                mv_mul_mv_conj = self * mv_conj
                numerator = (mv_conj * ~mv_mul_mv_conj)
                denominator = (self * numerator)[()]
```

Reply:
Fewer operations too - 1 fewer `gradeInvol` than your original.